_includes/getting-started.html

```diff
@@ -1,106 +1,117 @@
-        <label for="windows" data-tooltip1="Windows User"><img id="bounce" src="https://pngimg.com/uploads/windows_logos/windows_logos_PNG31.png" alt="" style="height: 50px;width: 50px;margin-left: 60px;top: -81px;position: relative;cursor:pointer"></label>
-        </div>
+      <h2><strong>Install and Start</strong></h2>
+      <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>01</strong></h1>
+
+
+      <div class="platformIcon">
+        <label for="start-helm-v2" data-tooltip="Bash User"><img id="bounce"
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4b/Bash_Logo_Colored.svg/1024px-Bash_Logo_Colored.svg.png"
+            alt=""></label>
+        <label for="start-brew" data-tooltip="Brew User"><img id="bounce"
+            src="https://brew.sh/assets/img/homebrew-256x256.png" alt=""></label>
+        <label for="k8s-instructions" data-tooltip="Kubernetes User"><img id="bounce"
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Kubernetes_logo_without_workmark.svg/1200px-Kubernetes_logo_without_workmark.svg.png"
+            alt=""></label>
+        <label for="windows" data-tooltip="Windows User"><img id="bounce"
+            src="https://pngimg.com/uploads/windows_logos/windows_logos_PNG31.png" alt=""></label>
+
+      </div>
 
-      
-        
-        <!-----------try---------->
-        <!-- Docker -->
-        <div class="toggle" style="margin-top: -200px;">
-          <label for="start-helm-v2" style="margin-left:1.8em;display: inline;	font-size: 1.38em;
+
+
+
+      <!-- Docker -->
+      <div class="toggle" style="margin-top: -200px;">
+        <label for="start-helm-v2" style="margin-left:1.8em;display: inline;	font-size: 1.38em;
           color: #666;
           font-weight: 300;
           line-height: 1.4em;">
-            <a></a>.</label>
-          <input type="checkbox" class="k8s-instructions" checked name="one" id="start-helm-v2">
-          <div class="hidediv" >
-            <div class="innerdiv" >
-              <p>Install on Mac or Linux using Docker:</p>
-  
-              <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
-                <div style="width:auto; height:30px; padding:0px; position:relative;border-radius: 5px;">
-                  <pre><code>   $ curl -L https://git.io/meshery | bash -
+          <a></a>.</label>
+        <input type="checkbox" class="k8s-instructions" checked name="one" id="start-helm-v2">
+        <div class="hidediv">
+          <div class="innerdiv">
+            <p>Install on Mac or Linux using Docker:</p>
+
+            <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+              <div style="width:auto; height:30px; padding:0px; position:relative;border-radius: 5px;">
```

Comment:
@gunjanmimo Can you check for margins over here? Although you will have to fix margin and padding issue in parent elements as well. It is causing following issues:-

Desktop view:-
Content is overflowing in x direction
![Screenshot from 2020-06-16 12-11-03](https://user-images.githubusercontent.com/35156299/84740666-6c64f900-afcb-11ea-98a9-0f6cd89ed048.png)